### DISCUSSION TITLE
MIDI performance

### DISCUSSION THREAD

**Discussion by bsjdsp** [apohl79/audiogridder#591] [General]
I’m curious to how using virtual instruments perform, doesn’t gridder add latency so even if I have lower latency for my daw when using virtual instruments, does the latency gridder add is still low enough to be acceptable

**Comment by dhstsw** [comment]
it depends of the buffer you allocate.
Overall, it's ininfluent with low buffers.

**Reply by Fittkenny** [reply]
ininfluent?????

Translatore per favore

**Reply by dhstsw** [reply]
I have about 22ms latency.
Meaning, irrelevant.

**Reply by Fittkenny** [reply]
22 ms is way to much to be playable
